### PR TITLE
Fixed title case for PL left nav

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/3_Authentication/2-two-factor-authentication.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Two-Factor Authentication
+title: Two-Factor authentication
```
Comment (roshnisarangadharan) [inline docs/platform/3_Authentication/2-two-factor-authentication.md:2]: Two-factor authentication

File: docs/platform/7_Connectors/Artifact-Repositories/connect-to-an-artifact-repo.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to an Artifact Repo
+title: Connect to an Artifact repo
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Artifact-Repositories/connect-to-an-artifact-repo.md:2]: repository

File: docs/platform/7_Connectors/Code-Repositories/connect-to-a-azure-repo.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to Azure Repos
+title: Connect to Azure repos
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Code-Repositories/connect-to-a-azure-repo.md:2]: Azure Repos (R should be capitalized)

File: docs/platform/7_Connectors/Code-Repositories/connect-to-code-repo.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to a Git Repo
+title: Connect to a Git repo
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Code-Repositories/connect-to-code-repo.md:2]: repository

File: docs/platform/7_Connectors/Code-Repositories/ref-source-repo-provider/http-helm-repo-connector-settings-reference.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: HTTP Helm Repo Connector Settings Reference
+title: HTTP Helm repo connector settings reference
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Code-Repositories/ref-source-repo-provider/http-helm-repo-connector-settings-reference.md:2]: repository (global) except for Azure Repos

File: docs/platform/Governance/14_Policy-as-code/add-a-policy-engine-step-to-a-secret.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Use Harness Policy As Code For Secrets
+title: Use Harness policy as code for secrets
```
Comment (roshnisarangadharan) [inline docs/platform/Governance/14_Policy-as-code/add-a-policy-engine-step-to-a-secret.md:2]: Shouldn't it be capitalized? Policy-as-Code or Policy as Code

File: docs/platform/Governance/14_Policy-as-code/using-harness-policy-engine-for-feature-flags.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Use Harness Policy As Code for Feature Flags
+title: Use Harness Policy As Code for feature flags
```
Comment (roshnisarangadharan) [inline docs/platform/Governance/14_Policy-as-code/using-harness-policy-engine-for-feature-flags.md:2]: Please make it Use Harness Policy as Code for Harness Feature Flags. We are referring to the module here. Lower case when we use flags.

**Review by roshnisarangadharan — DISMISSED** [review]
Approved with comments

File: docs/platform/Governance/14_Policy-as-code/add-a-policy-engine-step-to-a-secret.md (hunk 6 above)
Comment (rashminandasahoo) [reply docs/platform/Governance/14_Policy-as-code/add-a-policy-engine-step-to-a-secret.md:2]: It is Policy as Code.

File: docs/platform/Governance/14_Policy-as-code/using-harness-policy-engine-for-feature-flags.md (hunk 7 above)
Comment (rashminandasahoo) [reply docs/platform/Governance/14_Policy-as-code/using-harness-policy-engine-for-feature-flags.md:2]: Like connectors and secrets, here also it is implicit that we are referring to Harness Feature Flags.

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**